exporter/awsxrayexporter/translator/segment.go

```diff
@@ -80,10 +80,11 @@ func MakeSegmentDocumentString(span pdata.Span, resource pdata.Resource, indexed
 
 // MakeSegment converts an OpenTelemetry Span to an X-Ray Segment
 func MakeSegment(span pdata.Span, resource pdata.Resource, indexedAttrs []string, indexAllAttrs bool) (*awsxray.Segment, error) {
-	var segmentType string
+	var segmentType string = "segment"
```

Comment:
```suggestion
	var segmentType string
```

We only need to set the type for subsegments